examples/surrounding_nitrogen_bonds/n_n_fingerprint_funcs.py

```diff
@@ -0,0 +1,50 @@
+"""
+Various functions to create fingerprints for single nitrogen-nitrogen bonds
+"""
+
+from openeye import oechem, oeomega, oequacpac
+
+def find_neighboring_atoms(mol: oechem.OEMol, match: oechem.OEMatchBaseIter):
+    """
+    Finds and returns the neighbor atoms of a nitrogen-nitrogen bond as a list
+    given a match
```

Comment:
Can you comment on whether the fingerprint will always have the same length?

Reply:
If they are not the same length, that could have implications on what the results mean